template/template.go

```diff
@@ -202,12 +203,28 @@ func NewTemplateExpander(
 				}
 				return fmt.Sprintf("%.4g%s", v, prefix)
 			},
-			"humanizeDuration": func(v float64) string {
+			"humanizeDuration": func(i interface{}) (string, error) {
+				var (
+					v   float64
+					err error
+				)
+
+				s, ok := i.(string)
```

Comment:
we should use a switch here

Reply:
Yes, sure, this needs to get refined a lot. I just hacked something together to show how it works in principle.

If we do this, I'd actually prefer if someone else could do the nitty-gritty. I'm pretty busy at the moment with other things.